man/loader.conf.xml

```diff
@@ -215,6 +215,28 @@
         Currently, only x86 is supported, where it uses the PC speaker.</para></listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term>secure-boot-enrollment</term>
+
+        <listitem><para>Danger: this feature might soft-brick your device if used improperly.</para>
+
+        <para>Takes one of <literal>no</literal>, <literal>manual</literal>, <literal>auto</literal>
+        or <literal>force</literal>. Controls the enrollment of secure boot keys. If set to
+        <literal>no</literal> (the default), no action whatsoever is taken. If set to
+        <literal>manual</literal> and the UEFI firmware is in setup-mode then entries to manually enroll
+        Secure Boot keys are created in the boot menu. If set to <literal>auto</literal>, in addition, if a
+        directory named <filename>/loader/keys/auto</filename> exists on the ESP and there is a way to enroll
+        them safely (there is none for now), then the keys in that directory are enrolled. If set to
+        <literal>force<literal> then the keys in the auto directory are loaded, safely or not. This is
```

Comment:
I think "manual" should be the default. dropping in the keys + enabling the setup mode in the firmware setup should be enough to make these items appear.

and just drop "auto" for now. There's no point in adding something now that isn't implemented. Nothing stops us from adding "auto" later when the audit mode stuff is ready.